Makefile

```diff
@@ -152,12 +150,12 @@ endif
 	if [ $$(docker images | grep -c '<none>') -gt 0 ]; then \
 		docker images | grep '<none>' | awk '{print $$3;}' | xargs docker rmi; \
 	fi && \
-	reana-cluster -f ${PWD}/../reana-cluster/reana_cluster/configurations/reana-cluster-minikube$(addprefix -, ${CLUSTER_CONFIG}).yaml $(CLUSTER_FLAGS) init && \
+	helm install reana helm/reana-stable $(addprefix --set , ${CLUSTER_FLAGS}) --wait && \
```

Comment:
`s/reana-stable/reana/g` everywhere, and we'll get to releasing stable charts later using something like [chartpress](https://github.com/jupyterhub/chartpress).